### PR TITLE
Initial commit to create knative-eventing-extension

### PR DESCRIPTION
**Pull request by skaegi**
see https://github.com/tektoncd/community/issues/10

### CODE REVIEW

File: knative-eventing-extension/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# experimental
+Knative Eventing Extension
```
Comment (bobcatfish) [inline knative-eventing-extension/README.md]: is it possible to add some more initial information on what one would expect to find in this folder and what the plans are?

Comment (skaegi) [reply knative-eventing-extension/README.md]: You bet! I've added in a more complete project intro and tied into the community page.

Comment (bobcatfish) [reply knative-eventing-extension/README.md]: excellent! thanks @skaegi :heart: :D

**Pull request merged**